Casks/v/vimcal.rb

```diff
@@ -0,0 +1,24 @@
+cask "vimcal" do
+  arch arm: "-arm64", intel: ""
```

Comment:
```suggestion
  arch arm: "-arm64"
```

The `intel` part can be omittedhere.

Reply:
Resolved in https://github.com/Homebrew/homebrew-cask/pull/161467/commits/5a187c26b07c0103540c18cf573fe479278a8020